sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -433,9 +433,8 @@ object GpuOverrides {
     "when enabling these, there may be extra groups produced for floating point grouping " +
     "keys (e.g. -0.0, and 0.0)"
   val CASE_MODIFICATION_INCOMPAT =
-    "in some cases unicode characters change byte width when changing the case. The GPU string " +
-    "conversion does not support these characters. For a full list of unsupported characters " +
-    "see https://github.com/rapidsai/cudf/issues/3132"
+    "it will not be always 100% compatible with Spark because the unicode standards used by" +
+    " cuDF and JVM may differ."
```

Comment:
No need to explain it's incompatible, that is already covered elsewhere and redundant to put here.  This should be simplified to something like:
```suggestion
    "the Unicode standards used by cuDF and the JVM may differ, resulting in some " +
    "corner-case characters not changing case correctly."
```